tests/python/relay/test_pass_fuse_ops.py

```diff
@@ -199,42 +197,6 @@ def expected(dshape):
     assert relay.ir_pass.alpha_equal(zz, after)
 
 
-def test_tuple_strided_slice():
```

Comment:
this test case is removed, because now this is basically the same test case above test.